### PR TITLE
Add page @ /check-not-needed-location

### PR DESCRIPTION
**Pull request by JoeChapman**
An exit page used to inform the user its not necessary for them to apply.

### CODE REVIEW

File: apps/right-to-rent-check/index.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ module.exports = {
         }
       }]
     },
-    '/check-not-needed-location': {},
+    '/check-not-needed-location': {
+
```
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/index.js]: Did you mean this new line?

Comment (JoeChapman) [reply apps/right-to-rent-check/index.js]: yes I did. I just find it easier to read

Comment (sulthan-ahmed) [reply apps/right-to-rent-check/index.js]: hmm, do we need the extra blank line could we not do instead?
```
check-not-needed-location: {
}
```

**Pull request merged**